torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -6,13 +6,50 @@
 #include <torch/csrc/jit/passes/dead_code_elimination.h>
 #include <torch/csrc/jit/passes/graph_rewrite_helper.h>
 #include <torch/csrc/jit/passes/mkldnn_rewrite.h>
+#include <torch/csrc/jit/passes/mkldnn_rewrite_helper.h>
 #include <torch/csrc/jit/tensorexpr/kernel.h>
 
 namespace torch {
 namespace jit {
 
 #if AT_MKLDNN_ENABLED()
 
+namespace mkldnn {
+
+const std::vector<std::string> zero_scalar_operand =
+    std::vector<std::string>({});
+const std::string algorithm_indicator = std::string("%algorithm");
+
+std::string construct_operand_list(
+    std::vector<std::string> scalar_input,
+    std::string algorithm_indicator) {
+  std::string constructed_operand_list = "";
+
+  std::string joined_scalar_operands = c10::Join(", ", scalar_input);
+  std::string scalar_operands = "%scalars : Scalar?[] = prim::ListConstruct(" +
+      joined_scalar_operands + ")\n";
+
+  constructed_operand_list += scalar_operands;
+
+  if (algorithm_indicator.empty()) {
+    std::string algorithm_placeholder = "%algorithm : str? = prim::Constant()";
+    constructed_operand_list += algorithm_placeholder;
+  }
+
+  return constructed_operand_list;
+}
+
+const std::map<std::string, PostOp>& fusion_rewrite_map() {
```

Comment:
Please add a comment explaining what we're storing in this map.

Reply:
Yes, a comment is added.

Reply:
I still don't understand the reason why we need this map. It looks like some internal details exposed globally?

Reply:
fusion_rewrite_map definition is removed for your code readable.